colly.go

```diff
@@ -267,6 +268,11 @@ func (c *Collector) DisableCookies() {
 	c.backend.Client.Jar = nil
 }
 
+// SetClientTimeout overrides the default timeout for this collector
+func (c *Collector) SetClientTimeout(timeout time.Duration) {
```

Comment:
What about naming this function simply `SetTimeout` or maybe `SetRequestTimeout`?

Reply:
I actually named it SetTimeout at first but after writing the syntax out it kinda hit me that Collector.SetTimeout() kinda feels like you are setting the timeout on the collector itself and not the outgoing requests. I'll change it to SetRequestTimeout which makes more sense than SetClientTimeout